plugins/modules/win_ntp.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: 2020, Sakar Mehra (@sakar97) <sakar.mehra@gslab.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_ntp
+short_description: Manage Windows NTP Server
+author: Sakar Mehra (@sakar97)
+requirements:
+  - This module requires Windows Server 2012R2 or Newer
+description:
+  - Manage Windows NTP Client
+  - Adds, Removes and Modifies NTP Server in windows machine.
+options:
+  peerlist:
+    description:
+      - Fully qualified domain name or IP address from which computer obtain timestamp.
+    type: list
+  type:
+    description:
+      - Indicates which peers to accept synchronization from.
+      - The default value on stand-alone clients and servers is NTP.
+    type: str
+    choices: [ NTP, NoSync, NT5DS, AllSync ]
+  enabled:
+    description:
+      - Indicates if the NtpClient provider is enabled in the current Time Service.
+      - The default value on stand-alone clients and servers is true.
+    type: bool
+    default: true
+  time_zone:
+    description:
+      - Set the time zone for the ntp server.
+    type: str
+    default: UTC
+  factory_default:
+    description:
+      - Ntp configuration are set to default with this option.
+    type: bool
+    default: false
+  cross_site_sync_flags:
+    description:
+      - Determines whether the service chooses synchronization partners outside the domain of the computer.
+      - Thsi value is ignored if l(type=NT5DS) is not set.
+    type: int
+    choices: [ 0, 1, 2 ]
```

Comment:
Instead of using integer values you should use the string representations and convert them to the proper int values in the module. It's easier for people to understand what they are for if it's a string.